rgw/v2/tests/s3_swift/test_bucket_policy_ops.py

```diff
@@ -329,7 +389,10 @@ def test_exec(config, ssh_con):
         configure_logging(f_name=log_f_name, set_level=args.log_level.upper())
         config = Config(yaml_file)
         config.read(ssh_con)
-        if config.test_ops.get("upload_type") == "multipart":
+        if (
```

Comment:
@hmaheswa 
Since there are only two possible values for upload_type,
Instead of checking condition for both, we can just go with the condition "if (config.test_ops.get("upload_type")):" ryt?

Reply:
done @ckulal